.github/dependabot.yml

```diff
@@ -0,0 +1,11 @@
+# To get started with Dependabot version updates, you'll need to specify which
+# package ecosystems to update and where the package manifests are located.
+# Please see the documentation for all configuration options:
+# https://help.github.com/github/administering-a-repository/configuration-options-for-dependency-updates
+
+version: 2
+updates:
+  - package-ecosystem: "mix" # See documentation for possible values
```

Comment:
should we add `npm` as well for updates to js packages, like the last one for lodash?

Reply:
I was unsure, we could but that might get _very_ noisy... 🤔

Reply:
We may also be able to rely on [Phoenix keeping those sufficiently up to date for us](https://github.com/phoenixframework/phoenix/pull/3932).

Reply:
works for me and we can revisit later